tensorflow/python/ops/math_ops.py

```diff
@@ -386,6 +386,27 @@ def erf(x, name=None):
     else:
       return gen_math_ops.erf(x, name=name)
 
+    
+@tf_export("math.erfinv", "erfinv")
```

Comment:
Looks like the one in distributions/special_math.py currently isn't exported to public API.
Since it is useful, makes sense to add tf_export decorator to it, but I don't know about its implementation.

Looks like Eugene approved PR that added erfinv to special_math.py. @ebrevdo Eugene, can you take a look at the difference between 2 erfinv implementations and whether it makes sense to add the one in special_math.py to public API?

Reply:
Fyi, PR that added erfinv to special_math.py: https://github.com/tensorflow/tensorflow/pull/14056